files/build_templates/sonic_debian_extension.j2

```diff
@@ -116,6 +116,12 @@ sudo cp {{platform_common_py2_wheel_path}} $FILESYSTEM_ROOT/$PLATFORM_COMMON_PY2
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install $PLATFORM_COMMON_PY2_WHEEL_NAME
 sudo rm -rf $FILESYSTEM_ROOT/$PLATFORM_COMMON_PY2_WHEEL_NAME
 
+# Install sonic-daemon-base Python 2 package
+DAEMON_BASE_PY2_WHEEL_NAME=$(basename {{daemon_base_py2_wheel_path}})
+sudo cp {{daemon_base_py2_wheel_path}} $FILESYSTEM_ROOT/$DAEMON_BASE_PY2_WHEEL_NAME
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install $DAEMON_BASE_PY2_WHEEL_NAME
+sudo rm -rf $FILESYSTEM_ROOT/$DAEMON_BASE_PY2_WHEEL_NAME
```

Comment:
@stephenxs / @andriymoroz-mlnx this is code duplication. I think we need to improve the build system.

Reply:
> @stephenxs / @andriymoroz-mlnx this is code duplication. I think we need to improve the build system.

Do you mean we should extract these python-wheel installations into a common rule?
I admit it's a better way to do so but I think it's better to add it to our backlog and raise another PR when we have time to do it.